### DISCUSSION TITLE
OLED MT12 Mod

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#5728] [Q&A]
Hi all! I just have a quick question. (Maybe this isn’t really the right place to ask, but I’ll still try!)

I’m considering doing the OLED mod for the RadioMaster MT12. If I do that, what do I have to do to actually get it working with a new EdgeTX versions? For example, is there a certain modification I have to do to the code each time I update? Or will it just work automatically?

I know there was another repo with the code for the mod, however, I’m just assuming it would brick the screen if I ever updated to another standard version of Edge. 

_TLDR - if I do the OLED mod on my MT12, is it easy to keep it working correctly even with newer Edge versions?_
